common/theme.c

```diff
@@ -261,6 +263,39 @@ void load_theme(struct theme_config *theme, struct mux_config *config, struct mu
     theme->MUX.ITEM.PREV_HIGH = theme->MUX.ITEM.COUNT / 2 + 1;
     theme->MUX.ITEM.NEXT_LOW = theme->MUX.ITEM.COUNT / 2;
     theme->MUX.ITEM.NEXT_HIGH = theme->MUX.ITEM.COUNT / 2 + 1;
+    if (theme->MUX.ITEM.COUNT % 2 == 0)  {
```

Comment:
temporary fix to fix scrolling when menu item count is set to even number.  May look into refactoring scrolling in another PR